test/index.test.js

```diff
@@ -172,6 +183,27 @@ describe("iw-mysql-wrapper", function () {
 						]);
 					});
 				}).then(function verifyAfterCommit() {
+					expect(verifiedBeforeCommit).to.be.eql(true);
+					return Q.all([
+						savedScope.query("SELECT 1;").then(neverCallMe("Transaction should be closed.")).catch(function (err) {
+							expect(err.code).to.eql("E_TRANSACTION_CLOSED");
+						}),
+						db.query("SELECT * FROM `iw_mysql_wrapper_test`").then(function (rows) {
+							expect(rows).to.eql([{id: 1}, {id: 2}], "DB scope: should have values in the table (transaction is complete)");
+						})
+					]);
+				});
+
+			});
+
+			it("should execute multiple queries (without returning a promise)", function () {
```

Comment:
Maybe something like "should execute multiple queries within the same promise" would define the test better?